Source/Core/DolphinQt/MainWindow.ui

```diff
@@ -199,6 +197,31 @@
     <string>&amp;Icon view</string>
    </property>
   </action>
+  <action name="actionBrowse">
+   <property name="text">
+    <string>&amp;Browse for ISOs...</string>
+   </property>
+  </action>
+  <action name="actionExit">
+   <property name="text">
+    <string>Exit</string>
+   </property>
+  </action>
+  <action name="actionPlay_mnu">
```

Comment:
Why not just make these say "menu"? There's no need to shorthand a 4 letter word (unless Qt restricts that for some reason).